File: src/java/org/codehaus/testdox/intellij/actions/CloseQuickDoxAction.java
package org.codehaus.testdox.intellij.actions;

import com.intellij.openapi.actionSystem.AnActionEvent;
import org.codehaus.testdox.intellij.IconHelper;

public class CloseQuickDoxAction extends BaseAction {

    public CloseQuickDoxAction() {
        super("Close QuickDox", "Closes the TestDox tooltip", IconHelper.getIcon(IconHelper.DOX_ICON));
    }

    public void actionPerformed(AnActionEvent event) {
        actionEvents.getTestDoxController(event).closeQuickDox();
    }
}

File: src/main/scala/org/codehaus/testdox/intellij/actions/CloseQuickDoxAction.scala
package org.codehaus.testdox.intellij.actions

import com.intellij.openapi.actionSystem.AnActionEvent
import org.codehaus.testdox.intellij.Icons
import org.codehaus.testdox.intellij.Icons._

class CloseQuickDoxAction extends BaseAction("Close QuickDox", "Closes the TestDox tooltip", getIcon(Icons.DOX_ICON)) {

  def actionPerformed(event: AnActionEvent) {
    actionEvents.getTestDoxController(event).closeQuickDox()
  }
}
